src/consensus/params.h

```diff
@@ -29,6 +29,8 @@ struct BIP9Deployment {
     int64_t nStartTime;
     /** Timeout/expiry MedianTime for the deployment attempt. */
     int64_t nTimeout;
+    /** If lock in occurs, delay activation until at least this time. */
+    int64_t activation_time;
```

Comment:
We could add `reduced_threshold` which, if non-zero, overrides `nRuleChangeActivationThreshold`.

Reply:
I think we can worry about genericity if we ever need to add 2 simultaneous vbit releases... which I've heard rumor not too many people think we'd be likely to ever do that (although I think it is fine)

Reply:
IIUC with the latest change, `nRuleChangeActivationThreshold` is only used for the "the might be an unknown rule change" warning system. Each deployment now has its own threshold.